README.md

```diff
@@ -26,6 +26,22 @@ $ cd dropbox-sdk-js
 $ npm install
 ```
 
+If you are creating an application in the Browser, you can include the SDK using a distribution from a CDN.
+
+For example if you wanted to use version 1.0.0 you could include the following line in the `<head>` tag of your page. We discourage importing the latest version of the SDK by default as major version bumps contain breaking changes and could break your application without warning.
```

Comment:
nit: Add comma after 1.0.0: "For example if you wanted to use version 1.0.0, you..."